funnel/views/project.py

```diff
@@ -717,8 +717,8 @@ def admin(self):
                 )
             return redirect(self.obj.url_for('admin'), code=303)
         return {
-            'profile': self.obj.profile,
-            'project': self.obj,
+            'profile': self.obj.profile.current_access(datasets=('primary', 'related')),
```

Comment:
This should be just `datasets=('primary',)` (note extra comma) without `related` as we don’t want to include other projects.